docs/conf.py

```diff
@@ -76,6 +76,10 @@
     "mqt": ("https://mqt.readthedocs.io/en/latest/", None),
 }
 
+myst_enable_extensions = [
+    "colon_fence",
```

Comment:
```suggestion
    "amsmath",
    "colon_fence",
    "substitution",
    "deflist",
    "dollarmath",
```
Might as well copy over the full list of extensions we use in other projects.